tailwind.config.js

```diff
@@ -32,15 +32,32 @@ module.exports = {
           900: "#0d1829",
         },
       },
+      gridTemplateColumns: {
+        // Simple 8 column grid
+        8: "repeat(8, minmax(125px, 1fr))",
+      },
     },
     screens: {
       sm: "640px",
+      // => @media (min-width: 640px) { ... }
+
+      maxsm: { min: "100px", max: "767px" },
+      // => @media (max-width: 767px) { ... }
+
       md: "768px",
-      lg: "1024px",
+      // => @media (min-width: 768px) { ... }
+
+      lg: "999px",
```

Comment:
Any reason to change the default?

Reply:
Yes, it was for the previous section, but it was abandoned. I came back to the default.